### DISCUSSION TITLE
Please, remove the justify text button

### DISCUSSION THREAD

**Discussion by acirujano** [uxbox/uxbox#513] [Ideas]
Since we do not justify texts on the web or apps for accessibility reasons.

**Comment by elhombretecla** [comment]
Hi Ana!!! 
Thank you so much for your comment. 
We strongly agree with you that justifying texts is a bad practice in terms of readability for the web. Still, this feature can still be useful for other purposes or media, like purely aesthetic compositions or print. We prefer to keep it available in order to cover as much corner cases as possible. However, we are planning a text tool rework in the near future and we will take into account your remark, we might reconsider its relevance on the interface.

**Reply by acirujano** [reply]
Hi @elhombretecla!
Thanks for your explanation. I understand it could be useful for non-responsive environments. I hope it is used responsibly :)
Congratulations on the project, I love it!
